target/communication/fi_rng_commands.py

```diff
@@ -0,0 +1,75 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+"""Communication interface for OpenTitan RNG FI framework.
+
+Communication with OpenTitan happens over the uJSON command interface.
+"""
+import json
+import time
+from typing import Optional
+
+
+class OTFIRng:
+    def __init__(self, target) -> None:
+        self.target = target
+
+    def _ujson_rng_cmd(self) -> None:
+        time.sleep(0.01)
+        self.target.write(json.dumps("RngFi").encode("ascii"))
+        time.sleep(0.01)
+
+    def init(self, test: str) -> None:
+        """ Initialize the RNG FI code on the chip.
+
+        Args:
+            test: The selected test.
+        Returns:
+            The device ID of the device.
+        """
+        # RngFi command.
+        self._ujson_rng_cmd()
+        # Init command.
+        time.sleep(0.01)
+        if "csrng" in test:
+            self.target.write(json.dumps("CsrngInit").encode("ascii"))
+        # Read back device ID from device.
+        return self.read_response(max_tries=30)
+
+    def csrng(self) -> None:
```

Comment:
Nitpick:
Change function name to `rng_csrng_bias`
(Prefix to match other OTFI classes and suffix to match testname.)